apps/dh/api-dh/source/DataHub.WebApi/GraphQL/GraphQLQuery.cs

```diff
@@ -163,7 +167,7 @@ public GraphQLQuery()
 
             Field<NonNullGraphType<ListGraphType<NonNullGraphType<SettlementReportType>>>>("settlementReports")
                 .Argument<ProcessTypeEnum>("processType")
-                .Argument<string[]>("gridAreaCodes", nullable: true)
+                .Argument<string[]>("gridAreaCodes", true)
```

Comment:
Suggestion: keep named param, so it is easier to see what is set to true

Reply:
Will do